packages/api/methods.ts

```diff
@@ -192,3 +192,27 @@ export function updateRule(rule) {
 export function deleteRule(id) {
   return send('api/rule-delete', { id });
 }
+
+export function getSchedules() {
+  return send('api/schedules-get');
+}
+
+export function createSchedule(schedule, conditions) {
+  return send('api/schedule-create', { schedule, conditions });
+}
+
+export function updateSchedule(schedule, conditions, resetNextDate) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏It's likely we'll want to add more options to this method later on. Which would lead to a long list or arguments. What if we changed the last argument to be an object with optional attributes?

```suggestion
export function updateSchedule(schedule, conditions, { resetNextDate }: { resetNextDate?: boolean } = {}) {
```